warehouse/tests/mart/transit_database/test_dim_provider_gtfs_data_has_same_regional_types.sql

```diff
@@ -0,0 +1,48 @@
+{{ config(store_failures = true) }}
```

Comment:
should this be `True`

Reply:
Copied it from one of the existing `validate_` payments tests which seems to be working as expected (I see persisted failures from those)